doc/replicasets.rst

```diff
@@ -0,0 +1,226 @@
+.. _cartridge-cli.replicasets:
+
+===============================================================================
+Setting up replicasets
+===============================================================================
+
+``cartridge replicasets`` command is used to set up replicasets on local
```

Comment:
`on development` -> `for development`?

Reply:
Doc moved to #412 and will be fixed separately.